core/themes/basis/css/component/breadcrumb.css

```diff
@@ -4,7 +4,7 @@
  */
 .breadcrumb {
   overflow: hidden;
-  margin: 0 0 1em;
```

Comment:
Because @quicksketch closed https://github.com/backdrop/backdrop-issues/issues/2191 and asked us to address that in this issue.